pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -44,44 +45,55 @@ func ImageLocalityPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *scheduler
 		return schedulerapi.HostPriority{}, fmt.Errorf("node not found")
 	}
 
-	sumSize := totalImageSize(nodeInfo, pod.Spec.Containers)
+	var score int
+	if priorityMeta, ok := meta.(*priorityMetadata); ok {
+		score = calculatePriority(sumImageScores(nodeInfo, pod.Spec.Containers, priorityMeta.totalNumNodes))
```

Comment:
I know that this predates your changes but it'd be better to include initContainers.

Reply:
@silveryfu and @resouer, could you guys work on a follow-up PR to address this?

Reply:
Thanks! Ref this comment: https://github.com/kubernetes/kubernetes/pull/65745#discussion_r201619222 There will be following PR for `initContainers` specifically.

Reply:
Ohh sorry, did not notice it. Thanks @resouer